mars_rover/spec/traveller_spec.rb

```diff
@@ -81,8 +81,8 @@
           it "should raise an error with an appropriate error message" do
 
               loc2 = Location.new(3,3,"W")
-              path2 = 'MMMMRMMRMRRM'
-              traveller2 = Traveller.new(loc2, path2)
+              direction2 = Direction.new('MMMMRMMRMRRM')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.